hax/hax/util.py

```diff
@@ -228,6 +228,11 @@ def _service_by_name(self, hostname: str, svc_name: str) -> Dict[str, Any]:
             f'No {svc_name!r} Consul service found at node {hostname!r}')
 
     def get_local_nodename(self) -> str:
+        """
+        Returns the logical name of the current node. This is the name that
+        Consul is aware of. In other words, whenever Consul references a node,
+        it will use the names that this function can return.
+        """
```

Comment:
In normal case - yes. But in the failover case - not quite true. In the failover case both hax connected to the same consul agent listening on the default localhost port - that's why the environment variable was introduced (because we could not always determine the nodename from consul).

Reply:
Maybe you can suggest a better text?

Reply:
I'm not sure any text is needed here. The code seems quite self-explanatory.

Reply:
Well, historically it was you @andriytk who explained me why this function should be used in this context (so node name is returned and that's the name that Consul is aware of). If I don't want other developers to repeat the same communication route, I think that text is needed.

And yes, a function without a docstring makes our Codacy unhappy 😄